src/mapport.h

```diff
@@ -12,7 +12,12 @@ static const bool DEFAULT_UPNP = USE_UPNP;
 static const bool DEFAULT_UPNP = false;
 #endif
 
-void StartMapPort();
+enum MapPortProto : unsigned int {
```

Comment:
It will require to define bitwise operators which are used here:
https://github.com/bitcoin/bitcoin/blob/d9ecda8fa0ebce5b6da972691b19f4425164ae43/src/mapport.cpp#L297-L304

Maybe in follow-up?

Reply:
I wish there was a C++ standard QFlags...